deploy/configs/rules.yml

```diff
@@ -0,0 +1,28 @@
+groups:
+- name: oracle_keeper
+  rules:
+  # Alert for any instance that is unreachable for >2 minutes.
+  - alert: InstanceDown
+    expr: up == 0
+    for: 2m
+    labels:
+      severity: critical
+    annotations:
+      summary: "Instance {{ $labels.instance }} down"
+      description: "{{ $labels.instance }} of job {{ $labels.job }} has been down for more than 2 minutes."
+  - alert: OracleDown
```

Comment:
Do I understand correctly that if I run oracle but not the keeper, I won't receive any notifications about keeper down?

Reply:
Yes, that's right, in the docker compose I can't set rules depending on some conditions, we need to think about how best to solve this, have any ideas? @tsudmi